lib/faraday_middleware/response/parse_json.rb

```diff
@@ -8,7 +8,7 @@ class ParseJson < ResponseMiddleware
     end
 
     define_parser do |body, parser_options|
-      ::JSON.parse(body, parser_options) unless body.strip.empty?
+      ::JSON.parse(body, parser_options || {}) unless body.strip.empty?
```

Comment:
I went with the simple default to an empty hash, as from checking other implementations it is fine to do.